src/psyclone/psyir/nodes/node.py

```diff
@@ -89,7 +92,7 @@ def colored(text, _):
 
         :param text: Text to return
         :type text: string
-        :param _: Fake argument, only required to match interface
+        :param _: Fake argument, only required to match interface \
```

Comment:
Full stop at end of sentence.

Reply:
Perhaps fix the above param as well (lower case and full stop).

Reply:
Done.

Reply:
There is an uppercase issue here but I've taken the liberty to fix.